src/components/Txt/index.tsx

```diff
@@ -48,24 +60,54 @@ export const align = style({
 	cssProperty: 'text-align',
 });
 
+export const Lopy = (props: ThemedTxtProps) =>
+	props.truncate &&
+	css`
+		cursor: pointer;
+	`;
+
+export const LopyWrapper = styled.span`
+	display: inline-block;
+`;
+
 const BaseTxt = styled.div<TxtProps>`
 	${align}
 	${monospace}
 	${whitespace}
-
+	${code}
 	${caps}
 	${bold}
 	${italic}
-
 	${truncate}
 `;
 
 const Factory = (tag?: string) => {
 	return asRendition<React.FunctionComponent<TxtProps>>((props: any) => {
-		return <BaseTxt as={tag} {...props} />;
+		return props.copy ? (
+			<Copy
+				content={props.children}
```

Comment:
Not sure why TS is not complaining here but you are passing a possible JSX element as a string property.  This is not allowed, or at least this is not correct. You should pass porps.copy that should be a string containing what we want to copy

Reply:
I understand. I think this is what Matt brought up in the flowdock discussion. I changed content to be props.copy.